google/cloud/bigtable/cluster.py

```diff
@@ -145,6 +160,8 @@ def _update_from_pb(self, cluster_pb):
         self.location_id = cluster_pb.location.split("/")[-1]
         self.serve_nodes = cluster_pb.serve_nodes
         self.default_storage_type = cluster_pb.default_storage_type
+        if cluster_pb.encryption_config:
```

Comment:
what does this default to if this condition is not met?

Reply:
Oops. If it's being called from the constructor, it will be None, but we can't assume that can we? I'll add an `else` here.

Reply:
f1aae9d

Reply:
oh sorry missed that, makes sense thanks!

Reply:
Actually, you correctly pointed out a logical error, which I corrected. Thank you!